.cspell.json

```diff
@@ -0,0 +1,8 @@
+{
+	"version": "0.1",
+	"language": "en",
+	"dictionaries": ["en_US", "softwareTerms", "exonum"],
+	"dictionaryDefinitions": [
+		{ "name": "exonum", "path": "./exonum-dictionary.txt" }
```

Comment:
Perhaps we'll be able to share the same dictionary between multiple repos somehow. Just a side note.

Reply:
We can store in https://gist.github.com/ and fetch via `curl`.